src/main/scala/beam/sim/config/BeamExecutionConfig.scala

```diff
@@ -2,4 +2,17 @@ package beam.sim.config
 
 import org.matsim.core.config.{Config => MatsimConfig}
 
-case class BeamExecutionConfig(beamConfig: BeamConfig, matsimConfig: MatsimConfig, outputDirectory: String)
+case class BeamExecutionConfig(beamConfig: BeamConfig, matsimConfig: MatsimConfig, outputDirectory: String) {
+  import BeamExecutionConfig._
+
+  val executionId: String = executionIdFromDirectory(outputDirectory)
+
+}
+
+object BeamExecutionConfig {
+
```

Comment:
Why is this in an object? I'd default to keeping it internal to the case class until it is needed external